src/test/java/uk/gov/pay/adminusers/resources/InviteUserRequestValidatorTest.java

```diff
@@ -275,4 +278,53 @@ public void validateOtpValidationRequest_shouldError_ifOtpFieldIsMissing() throw
         assertThat(errors.getErrors(), hasItems(
                 "Field [otp] is required"));
     }
+
+    @Test
+    public void shouldSuccess_ifAllFieldsArePresentAndValidEmailDomain() throws Exception {
+        ImmutableMap<String, String> payload = ImmutableMap.of("email", "example@example.gov.uk", "telephone_number", "08000001066", "password", "super-secure-password");
+        JsonNode payloadNode = objectMapper.valueToTree(payload);
+        Optional<Errors> errors = validator.validateCreateServiceRequest(payloadNode);
+
+        assertFalse(errors.isPresent());
+    }
+
+    @Test
+    public void shouldFail_ifMissingRequiredField() throws Exception {
+        ImmutableMap<String, String> payload = ImmutableMap.of( "telephone_number", "08000001066", "password", "super-secure-password");
+        JsonNode payloadNode = objectMapper.valueToTree(payload);
+        Optional<Errors> errors = validator.validateCreateServiceRequest(payloadNode);
+
+        assertTrue(errors.isPresent());
+        assertThat(errors.get().getErrors(),hasItems("Field [email] is required"));
+        assertThat(errors.get().getErrors().size(),is(1));
+    }
+
+    @Test
+    public void shouldFail_ifInvalidEmailFormat() throws Exception {
+        ImmutableMap<String, String> payload = ImmutableMap.of( "email", "exampleatexample.com", "telephone_number", "08000001066", "password", "super-secure-password");
+        JsonNode payloadNode = objectMapper.valueToTree(payload);
+        Optional<Errors> errors = validator.validateCreateServiceRequest(payloadNode);
+
+        assertTrue(errors.isPresent());
+        assertThat(errors.get().getErrors(),hasItems("Field [email] must be a valid email address"));
+        assertThat(errors.get().getErrors().size(),is(1));
+    }
+
+    @Test(expected = WebApplicationException.class)
+    public void shouldFail_ifEmailAddressNotPublicSector() throws Exception {
+        ImmutableMap<String, String> payload = ImmutableMap.of( "email", "example@example.co.uk","telephone_number", "08000001066", "password", "super-secure-password");
+        JsonNode payloadNode = objectMapper.valueToTree(payload);
+        validator.validateCreateServiceRequest(payloadNode);
+    }
+
+    @Test
+    public void shouldFail_ifInvalidTelephoneNumber() throws Exception {
```

Comment:
I would probably create one for a number with less than 11 digits and another for a telephone number with correct number of digits but containing a letter